.build/jest/test-results.json

```diff
@@ -0,0 +1 @@
s","location":null,"status":"passed","title":"Renders correctly with defaults"},{"ancestorTitles":["ProgressCircular"],"failureMessages":[],"fullName":"ProgressCircular Renders correctly with property variants","location":null,"status":"passed","title":"Renders correctly with property variants"}],"endTime":1588692168754,"message":"","name":"/Users/mark.johnson/code/components/src/components/Progress/components/ProgressCircular/test.tsx","startTime":1588692162814,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":["ProgressLinear"],"failureMessages":[],"fullName":"ProgressLinear Renders correctly with defaults","location":null,"status":"passed","title":"Renders correctly with defaults"},{"ancestorTitles":["ProgressLinear"],"failureMessages":[],"fullName":"ProgressLinear Renders correctly with property variants","location":null,"status":"passed","title":"Renders correctly with property variants"}],"endTime":1588692168754,"message":"","name":"/Users/mark.johnson/code/components/src/components/Progress/components/ProgressLinear/test.tsx","startTime":1588692162814,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":[],"failureMessages":[],"fullName":"renders correctly","location":null,"status":"passed","title":"renders correctly"},{"ancestorTitles":[],"failureMessages":[],"fullName":"renders correctly with various props","location":null,"status":"passed","title":"renders correctly with various props"}],"endTime":1588692169616,"message":"","name":"/Users/mark.johnson/code/components/src/components/Flag/test.tsx","startTime":1588692162814,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":[],"failureMessages":[],"fullName":"renders correctly","location":null,"status":"passed","title":"renders correctly"},{"ancestorTitles":[],"failureMessages":[],"fullName":"renders correctly with various props","location":null,"status":"passed","title":"renders correctly with various props"}],"endTime":1588692169756,"message":"","name":"/Users/mark.johnson/code/components/src/components/Typography/test.tsx","startTime":1588692162816,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":[],"failureMessages":[],"fullName":"renders correctly","location":null,"status":"passed","title":"renders correctly"},{"ancestorTitles":[],"failureMessages":[],"fullName":"renders correctly with various props","location":null,"status":"passed","title":"renders correctly with various props"}],"endTime":1588692169941,"message":"","name":"/Users/mark.johnson/code/components/src/components/Button/test.tsx","startTime":1588692162814,"status":"passed","summary":""},{"assertionResults":[{"ancestorTitles":[],"failureMessages":[],"fullName":"renders correctly","location":null,"status":"passed","title":"renders correctly"}],"endTime":1588692170004,"message":"","name":"/Users/mark.johnson/code/components/src/components/ExpansionPanel/test.tsx","startTime":1588692162815,"status":"passed","summary":""}],"wasInterrupted":false}
```

Comment:
I think this would be better solved by adding this file to the project's `.gitignore`.